x-pack/plugins/apm/public/components/app/infra_overview/infra_tabs/index.tsx

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiTabbedContent,
+  EuiTabbedContentProps,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import type { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
+import { useApmServiceContext } from '../../../../context/apm_service/use_apm_service_context';
+import { useApmParams } from '../../../../hooks/use_apm_params';
+import { useTimeRange } from '../../../../hooks/use_time_range';
+import { FETCH_STATUS, useFetcher } from '../../../../hooks/use_fetcher';
+import { ApmPluginStartDeps } from '../../../../plugin';
+import { EmptyPrompt } from './empty_prompt';
+import { FailurePrompt } from './failure_prompt';
+
+type Tab = NonNullable<EuiTabbedContentProps['tabs']>[0] & {
+  id: 'containers' | 'pods' | 'hosts';
+  hidden?: boolean;
+};
+
+export function InfraTabs() {
+  const { serviceName } = useApmServiceContext();
+  const {
+    query: { environment, kuery, rangeFrom, rangeTo },
+  } = useApmParams('/services/{serviceName}/infra');
```

Comment:
Any reasons why we call this route `/services/{serviceName}/infra` instead of `/services/{serviceName}/infrastructure`?

Reply:
Because there's another route with this name, I'll take a look today to decide what to do with the previous infrastructure route

Reply:
Is that the server endpoint that you are referring to?

Reply:
I renamed the endpoints and I will open an issue to work on a follow-up to remove the logs endpoint